GVRf/Extensions/gvrf-physics/src/main/java/org/gearvrf/physics/GVRWorld.java

```diff
@@ -49,6 +51,8 @@ public GVRWorld(GVRContext gvrContext) {
     public GVRWorld(GVRContext gvrContext, GVRCollisionMatrix collisionMatrix) {
         super(gvrContext, NativePhysics3DWorld.ctor());
         mHasFrameCallback = false;
+        mIsProcessing = false;
```

Comment:
What is setting mIsProcessing to true? Are there changes missing from this pull request?